pkg/helm/helm.go

```diff
@@ -300,12 +304,18 @@ func (c *Client) Template(ctx context.Context, chart string, opts TemplateOption
 	// to set it ourselves.
 	client.ReleaseName = releaseName
 
+	previous := os.Getenv("HELM_CONFIG_HOME")
```

Comment:
We run a lot of our tests with `t.Parallel` so this isn't safe to do. It could lead to helm instances leaking into each other and accidentally touching the global helm config.

Reply:
agree